og/entity.go

```diff
@@ -316,6 +316,11 @@ type Filter struct {
 	Conditions         []Condition        `json:"conditions,omitempty"`
 }
 
+type IntegrationActionFilter struct {
+	ConditionMatchType ConditionMatchType `json:"conditionMatchType,omitempty"`
+	Conditions         []Condition        `json:"conditions,omitempty"`
+}
+
```

Comment:
You can define filter for integration in request.go instead of here since common things are defined in entity and filter for integration is special for integration.

Reply:
I have moved the integration-specific struct into integraion/request.go
Do let me know if any other change is necessary, thanks!